frontend/src/hooks/useInputMap.ts

```diff
@@ -1,22 +1,28 @@
+import AnyField from '@/components/global/fields/AnyField.vue';
 import StringField from '@/components/global/fields/StringField.vue';
 import IntegerField from '@/components/global/fields/IntegerField.vue';
 import BooleanField from '@/components/global/fields/BooleanField.vue';
+import type { ContractParamsSchema } from '@/types';
 
 export const InputTypesMap: { [k: string]: any } = {
   string: StringField,
   int: IntegerField,
   bool: BooleanField,
+  any: AnyField,
 };
 
 export const useInputMap = () => {
-  const getComponent = (type: string) => {
+  const getComponent = (type: ContractParamsSchema) => {
+    if (typeof type !== 'string') {
+      type = 'any';
+    }
     const component = InputTypesMap[type];
 
     if (!component) {
       console.warn(
-        `Component not found for input type: ${type}, defaulting to string`,
+        `Component not found for input type: ${type}, defaulting to any`,
       );
-      return StringField;
+      return AnyField;
     }
```

Comment:
Is "any" an explicit type of field coming from the vm?

Reply:
Yes, "any" is for fields that are not strings, booleans or ints. For instance it includes arrays and dictionaries